packages/curve-ui-kit/src/shared/ui/PositionDetails/tooltips/CollateralMetricTooltip.tsx

```diff
@@ -0,0 +1,81 @@
+import { Stack, Typography } from '@mui/material'
+import { FORMAT_OPTIONS, formatNumber } from '@ui/utils/utilsFormat'
+import { t } from '@ui-kit/lib/i18n'
+import { CollateralValue } from '@ui-kit/shared/ui/PositionDetails'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+
+const { Spacing } = SizesAndSpaces
+
+type CollateralMetricTooltipProps = {
+  collateralValue: CollateralValue | undefined | null
+}
+
+export const CollateralMetricTooltip = ({ collateralValue }: CollateralMetricTooltipProps) => {
+  const collateralValueFormatted =
```

Comment:
It's hard to understand what's going on with all this formatting, mainly when trying to see which ones are different and why. We might need some helpers with clear names